app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/js/view/shipping-information/address-renderer/store-pickup-address.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'underscore',
+    'uiComponent',
+    'Magento_Customer/js/customer-data',
+], function(_, Component, customerData) {
+    'use strict';
+
+    var countryData = customerData.get('directory-data');
+
+    return Component.extend({
+        defaults: {
+            pickupLocation: {},
+            template:
+                'Magento_InventoryInStorePickupFrontend/shipping-information/address-renderer/store-pickup-address',
+        },
+        initialize: function() {
+            this._super();
+            this.extractPickupLocationFromAddress(this.address());
+        },
+
+        initObservable: function() {
+            return this._super().observe(['pickupLocation']);
+        },
+
+        /**
+         * @param {*} countryId
+         * @return {String}
+         */
+        getCountryName: function(countryId) {
+            return countryData()[countryId] != undefined
```

Comment:
Use strict check `!==` if we really need to check value to `undefined` or don't use equal check at all ( `countryData()[countryId] ? .... : ....` if any false/true value is ok for you.

Reply:
or use _.isUndefined method